vm-allocator/src/system.rs

```diff
@@ -123,9 +144,10 @@ impl SystemAllocator {
         size: GuestUsize,
         align_size: Option<GuestUsize>,
     ) -> Option<GuestAddress> {
-        self.platform_mmio_address_space.allocate(
+        self.ram_address_space.allocate_in_region(
```

Comment:
An important piece of nomenclature here. From our perspective RAM is memory in the address space that is backed by RAM pages from the host. MMIO memory, 32-bit or 64-bit is not RAM as there is no backing of it to RAM pages. `virtio-pmem` is backed by pages but those are not stored in RAM on the host (unless you `mmap()` a file in /tmp)

Reply:
good point! maybe phys_address_space makes more sense